### ISSUE TITLE
Is it okay to keep large learning rate or change batch size while training is running?

### ISSUE THREAD

**Issue by StringBottle** [open-mmlab/mmdetection#1836]
I am training Mask R-CNN model with MobileNet-V3 as a backbone on COCO data-set (I am training the whole network from the scratch). I have got AP=0.203 at 12th epoch and still got AP=0.205 at 28th epoch. 

Since 15th epoch, the training loss doesn't decrease and two options were changed around 14-15th epoch. 

First, I had to change the batch size due to CUDA memory issue while training the model with 12 images per gpu. There was no memory issue before 15th epoch but it kept occurring after 15th epoch so I changed the batch size to 10 and resumed the training. 

Second, by configuration for learning rate schedule, the learning rate decreased to 0.00015.

Given the above observations, I think the two changes might be the cause of slow training loss decrease after 15th epoch. (Or there might be a bug in my code.)

I'd like to hear your opinions on this issue. 

Thanks in advance.

**Comment by StringBottle** [comment]
![MobileNet_Seg_14](https://user-images.githubusercontent.com/39578903/71172820-96e82a80-22a4-11ea-8978-88f5b3103c66.JPG)

**Comment by StringBottle** [comment]
![MobileNet_bbox_14](https://user-images.githubusercontent.com/39578903/71172845-a9626400-22a4-11ea-8864-287ffd7e14c2.JPG)

**Comment by StringBottle** [comment]
![캡처](https://user-images.githubusercontent.com/39578903/71172882-bed78e00-22a4-11ea-90cf-d74551fb0152.JPG)

**Comment by hellock** [comment]
We have not tried exactly the same thing and you may need to figure out the best hyper-parameters in your case. For a reference, you may refer to the Linear Scaling Rule in the [GETTING STARTED](https://github.com/open-mmlab/mmdetection/blob/master/docs/GETTING_STARTED.md#train-a-model).

**Comment by StringBottle** [comment]
@hellock 
Thanks for your reply. I will try with different parmenters.
